changelogs/fragments/10346-jenkins-plugins-fixes.yml

```diff
@@ -0,0 +1,6 @@
+bugfixes:
+  - "jenkins plugins plugin - install latest compatible version instead of latest (https://github.com/ansible-collections/community.general/pull/10346)."
```

Comment:
```suggestion
  - "jenkins_plugin - install latest compatible version instead of latest (https://github.com/ansible-collections/community.general/issues/854, https://github.com/ansible-collections/community.general/pull/10346)."
```